### PR TITLE
Guideline documentation contribution

### PR DESCRIPTION
**Pull request by RBernsen**
Added an additional section for how to contribute to the documentation

### CODE REVIEW

**Comment by RBernsen** [comment]
Hi @as-op ! Would you mind having a look over the new section from a technical point of view? I would then also give it to Birthe for review. Many thanks! Rebecca

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+- Guides: getting started guide, user guide, system admin guide, Enterprise edition guide, installation and operation guide. Thereby each guide is available at top level on its own.
+- FAQs
+- Release notes
+- Development
+- API
+
+Within each folder there is a sub-hierarchy of topics. E.g. in the Getting started guide you find amongst others the Introduction to OpenProject and Sign in and registration as sub-topics. Sub-topics are individual documentation pages in GitHub. You will notice the sub-topics in the documentation menu on the left when unfolding each folder's menu.
+
+![OpenProject_documentation_menu](OpenProject_documentation_menu.png)
```
Comment (as-op) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: Please provide a better ALT-Title for screen-readers (for humans).

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+## Directory and file names
+
+Please respect the following when working with directories and files:
+
+1. When you create a new topic, i.e. a new documentation page, always create a new folder and a new README.md file in that folder.
+2. Do not use special characters and spaces, or capital letters in file names, directory names, branch names and anything that generates a path.
+
+3. When creating a file or directory and it has more than one word in its name, use underscores (`_`) instead of spaces or dashes. For example use open_details-view_work_packages.png. This applies to both image files and Markdown files.
+
+1. For image files, do not exceed 200KB.
```
Comment (as-op) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: Should be bullet point nr. 4

**Review by as-op — APPROVED** [review]
👍 all info is correct, just some markdown trifles.

**Comment by HDinger** [comment]
Hi @RBernsen 
please keep in mind that changes that should be directly published need to be merged into the branch `release/11.4`. If you are unsure how to do that, I can help you, just give me a sign :)

File: docs/development/contribution-documentation/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+- Adding use cases.
+
+- Adding visuals, e.g. screenshots, to complement descriptions.
+
+- Fixing of spelling, grammar, punctuation mistakes.
+
+- Fixing of internal and external links that are not working or are incorrect.
+
+
```
Comment (birthe) [inline docs/development/contribution-documentation/README.md:54]: Maybe adding - Translations to your mother tongue (coming soon).

File: docs/development/contribution-documentation/contribution-support/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Login to or register at the [OpenProject community platform](https://community.openproject.org/login). It’s fast and free.
+
+2. Open a new [documentation work package]( https://community.openproject.org/projects/openproject/work_packages/new?type=69).
+
+3. Add a precise subject.
+
+4. Add a detailed description about your issue and how we could help.
+
+5. Attach a screen-shot or log-file file (optional).
```
Comment (birthe) [inline docs/development/contribution-documentation/contribution-support/README.md]: log-file (optional).

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+
+### UI text
+
+When referring to specific user interface text, like a button label or menu item, use the name as in the application and start the word with a capital letter. Moreover, please make it bold. Example: **Start free trial** button.
+
+
+
+### Feature names
+
+Feature names are typically capitalized and in bold. For example:
```
Comment (birthe) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: Only the first word is capitalized

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 6]
```diff
@@ -0,0 +1,428 @@
...
+
+When documenting navigation through the OpenProject application, use these terms.
+
+
+
+### Menus
+
+Use these terms when referring to OpenProject’s main application elements:
+
+- **Header menu**: This is the blue bar at the top that spans the width of the application. It includes the OpenProject logo, the search field, the link to all projects, the short menu, the help icon and the user’s avatar.
```
Comment (birthe) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: What do you mean by short menu? Global modules (the one with the dots)?

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 7]
```diff
@@ -0,0 +1,428 @@
...
+
+- **Project menu**: This is the black menu on the left in the OpenProject application that displays the modules.
```
Comment (birthe) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: Does not necessarily have to be black. Maybe just the menu on the left that appears after the selection of a project.

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 8]
```diff
@@ -0,0 +1,428 @@
...
+
+- Place your images in the same directory where the README.md document that you are working on is located.
+
+- Compress GIFs, maximum size 250KB.
+
+  
+
+### Add the image link to content
+
+The Markdown code for including an image in a document is: `![Image description which will be the alt tag](img/document_image_title_vX_Y.png)
```
Comment (birthe) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: Hier fehlt die Klammer zu ]

**Review by birthe — APPROVED** [review]

**Pull request merged**